crates/typed-store/src/rocks/mod.rs

```diff
@@ -99,11 +109,52 @@ pub struct DBMap<K, V> {
     db_metrics: Arc<DBMetrics>,
     read_sample_interval: SamplingInterval,
     write_sample_interval: SamplingInterval,
+    _sender: Arc<oneshot::Sender<()>>,
```

Comment:
Can we name this something a bit more descriptive of what its purpose is? Something like `_metrics_task_cancel_handle` or even a descriptive comment of its use would also suffice.